site/content/kapp-controller/docs/latest/private-registry-auth.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Authenticating to Private Registires
+---
+
+In order for kapp-controller to pull imgpkg bundles or images from registries requiring authentication, 
```

Comment:
nit,  but the sentence reads equally well if we strike the first few words,   and maybe better if 'images' comes first bc the "imgpkg" modifier only applies to bundles?: 

"To pull images or imgpkg bundles from registries requiring authentication, kapp-controller supports..."

Reply:
Thanks for the language brevity suggestion. I have added in. I am going to keep imgpkg bundles first since we like to subtly suggest imgpkg bundles over plain images. But if you still feel strongly on this, maybe we can find a way to edit further in a way that keeps bundles first.